### DISCUSSION TITLE
Support for embeddings with text-embedding-ada-002 (1536 tokens)

### DISCUSSION THREAD

**Discussion by ivankrsul** [betalgo/openai#124] [General]
It seems that the library only supports text-embedding-ada-001 (the TextSearchAdaDocV1 in the Models class) and the corresponding output dimensions is limited to 1024 tokens, while the text-embedding-ada-002 has 1536 tokens.  Any plans to upgrade the library to use text-embedding-ada-002?

Ivan

**Comment by kayhantolga** [comment]
I can add support on the next version. Also, you can always pass models as string.

**Comment by kayhantolga** [comment]
v6.7.2 supports new text-embedding-ada-002 model.
